glossary/glossary.adoc

```diff
@@ -9,6 +9,10 @@ Blue/Green deployment strategy:: In this deployment strategy two target environm
 Continuous value delivery:: Refers to the idea of delivering working software features at the end of each sprint in a scrum agile process
 Continuous deployment:: Refers to the process of automatically deploying software changes in a system as they occur instead of larger release cycles.
 Canary deployment strategy:: In this deployment strategy , some fraction of the deployment environment is updated with a new change. If all goes well, then the rest of the environment is updated. This allows for testing of experimental features in a production environment.
+Collaboration Space:: 
+* A container for related data within a defined scope. It is up to the creator of the collaboration space to define the scope. The scope can be very small (“this Collaboration Space is used to manage the development of HelloWorld”) or very large (“this Collaboration Space is used to manage the development of all of Apache OpenOffice”). A Collaboration Space has a name and a description. It may contain work item definitions and work items. It may contain Codebases. It may contain Workspaces. It may contain Pipeline Definitions. It may contain Teams. It contains at least one Area (“/” by default) and one Iteration. A Collaboration Space is created from a Process Template, which seeds the Collaboration Space with an initial set of Work Item Types and various other artifacts (e.g. an initial set of Iterations). Collaboration Spaces are either Public or Private. [Public collaboriation spaces permit anonymous viewing of all content within the collaboration space, although creation of content in the collaboration space may be controlled/limited by collaboration space administrators. Private collaboration spaces permit essentially any access model.]
```

Comment:
These sentences could be written as a list, I think:
> It may contain Codebases. It may contain Workspaces. It may contain Pipeline Definitions. It may contain Teams.